compiler/record-hessian/src/RecordHessian.cpp

```diff
@@ -0,0 +1,386 @@
+/*
```

Comment:
these are some questions,
- what is the purpose of this file? or what does this do?
- what are the tests for the codes in this file?